packages/back-end/src/scim/users/createUser.ts

```diff
@@ -6,30 +6,33 @@ import {
   expandOrgMembers,
 } from "../../services/organizations";
 import { OrganizationInterface } from "../../../types/organization";
-import {
-  ScimError,
-  ScimUser,
-  ScimUserPutOrPostRequest,
-} from "../../../types/scim";
+import { ScimError, ScimUser, ScimUserPostRequest } from "../../../types/scim";
 import {
   createUser as createNewUser,
   getUserByEmail,
 } from "../../services/users";
 
 export async function createUser(
-  req: ScimUserPutOrPostRequest,
+  req: ScimUserPostRequest,
   res: Response<ScimUser | ScimError>
 ) {
   const { externalId, displayName, userName } = req.body;
 
   const org: OrganizationInterface = req.organization;
 
+  let role = org.settings?.defaultRole?.role || "readonly";
+
+  const scimDefinedRole = req.body.growthbookRole;
```

Comment:
This currently doesn't guard against incorrect roles. E.G. A user could pass in `abc123` and that would be set as the user's role. Need to block that from happening.